api.go

```diff
@@ -305,6 +305,7 @@ func (chef *Chef) makeRequest(request *http.Request) (*http.Response, error) {
 	var client *http.Client
 	if chef.SSLNoVerify {
 		tr := &http.Transport{
+                        Proxy: http.ProxyFromEnvironment,
```

Comment:
Yeah I noticed that myself as well... Very sorry about that and don't know what went wrong (I use Sublime configured to call `goimports` on every save).

Will make a PR to fix this...